### DISCUSSION TITLE
Emoji use on Linux Web App Service

### DISCUSSION THREAD

**Discussion by DannyT** [OrchardCMS/OrchardCore#13529] [General]
Bit of an odd one. When using emoji in any fields/parts, the content item doesn't update, I get the feedback message telling me it was successful, but it isn't. If creating a new item, it doesn't create (nor show in audit trail) and if editing an existing item, the changes aren't saved.

This only happens on my prod environment which is an Azure Linux Web App service + MySQL database. When running locally (windows/SQLite) it works fine.

Is there some Linux-specific config I need to do?

**Comment by hishamco** [comment]
Could you please try it in MySQL locally, may be it's encoding issue

**Comment by dsghi** [comment]
FWIW - I can confirm this behavior: same setup, Azure Linux App service, and Azure Database for MySQL. The database character set is utf8, and the collation is utf8_general_ci. I also tried a markdown part with emoji support enabled, I can enter and save, but it vanishes somewhere along the line.
